src/typed.ts

```diff
@@ -1,48 +1,48 @@
-export const isSymbol = (value: any) => {
+export const isSymbol = (value: any): value is symbol => {
   return !!value && value.constructor === Symbol
 }
 
-export const isArray = (value: any) => {
+export const isArray = (value: any): value is any[] => {
```

Comment:
should we use `value is unknown[]` instead of `any`? I'm really not sure, been told a lot to use `unknown` over `any` when possible. What do you think?

Reply:
You are right, `unknown` is safer